tests/test_resource_realtimeapi.py

```diff
@@ -25,7 +25,7 @@ def _get_class_path(clsname):
         return '{}.{}.{}'.format(__package__, module_name, clsname)
 
     def test_realtimeapi_with_default_settings(self):
-        expected_entities = {'crawl.json': CrawlResource}
+        expected_entities = {b'crawl.json': CrawlResource}
```

Comment:
I'll just added to_bytes(route), so that before putChild route will be encoded to bytes.